charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -390,6 +390,8 @@ spec:
           name: reporting-operator-trusted-ca-bundle
           readOnly: true
 {{- end }}
+        - mountPath: /var/nfs
+          name: nfs-storage
```

Comment:
We don't need to mount anything in either of the containers in the reporting-operator Pod.